MonoGame.Framework.Content.Pipeline/OpenAssetImporter.cs

```diff
@@ -14,7 +14,46 @@
 
 namespace Microsoft.Xna.Framework.Content.Pipeline
 {
-    [ContentImporter(".dae", ".fbx", ".x", DisplayName = "Open Asset Import Library - MonoGame", DefaultProcessor = "ModelProcessor")]
+    [ContentImporter(
+        ".dae", // Collada
+        ".gltf", "glb", // glTF
+        ".blend", // Blender 3D
+        ".3ds", // 3ds Max 3DS
+        ".ase", // 3ds Max ASE
+        ".obj", // Wavefront Object
+        ".ifc", // Industry Foundation Classes (IFC/Step)
+        ".xgl", ".zgl", // XGL
+        ".ply", // Stanford Polygon Library
+        ".dxf", // AutoCAD DXF
+        ".lwo", // LightWave
+        ".lws", // LightWave Scene
+        ".lxo", // Modo
+        ".stl", // Stereolithography
+        ".ac", // AC3D
+        ".ms3d", // Milkshape 3D
+        ".cob", ".scn", // TrueSpace
+        ".bvh", // Biovision BVH
+        ".csm", // CharacterStudio Motion
+        ".xml", // Ogre XML
```

Comment:
This is another that we may want to remove...  else every time you add an XML file to your content project it will automatically decide it is a model file.